src/ui/src/abstractlogview.cpp

```diff
@@ -2425,7 +2420,7 @@ void AbstractLogView::drawTextArea( QPaintDevice* paintDevice )
             const auto lengthDelta
                 = static_cast<LineLength::UnderlyingType>( expandedMatchLength - matchPart.size() );
 
-            return HighlightedMatch{ match.startColumn() + startDelta, match.size() + lengthDelta,
```

Comment:
abstractlogview.cpp(2428): error C2398: Element '1': conversion from 'qsizetype' to 'int' requires a narrowing conversion
C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.32.31326\include\algorithm(3024): note: see reference to function template instantiation 'HighlightedMatch AbstractLogView::drawTextArea::<lambda_3>::operator ()<HighlightedMatch>(const _T1 &) const' being compiled